src/components/presentational/SingleExternalAccount/SingleExternalAccount.stories.tsx

```
@@ -1,79 +1,61 @@
+    },
+    render: render
+};
+
+export const Unauthorized: Story = {
+    args: {
+        externalAccount: { ...externalAccount, status: "unauthorized" }
+    },
+    render: render
+};
+
+export const Error: Story = {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)